app/views/terms_mailer/reminder_email.html.erb

```diff
@@ -2,13 +2,13 @@
 
 <p>You are receiving this email as a current or past user of the Stanford Digital Repository (SDR) self-deposit application at <a href="https://sdr.stanford.edu">https://sdr.stanford.edu</a>.</p>
 
-<p>All SDR depositors must agree to our Terms of Deposit, which assert that you have the right to deposit the content and assign Stanford University Libraries the necessary license to preserve the content and make it available online for others to access.</p>
+<p>All SDR depositors must agree to our <a href="https://sdr.library.stanford.edu/documentation/understanding-sdr-terms-deposit">Terms of Deposit</a>, which assert that you have the right to deposit the content and assign Stanford University Libraries the necessary license to preserve the content and make it available online for others to access.</p>
 
 <p>If you have deposited content in the SDR, you most likely agreed to the Terms of Deposit when you completed your first deposit. Your agreement to the Terms applies to all the deposits you make in the SDR, even though you may not be required to attest to the Terms every time you deposit content.</p>
 
 <span><strong>What should you do?</strong></span>
 
-<p>We ask that all SDR depositors review the Terms of Deposit on an annual basis. Learn more about the Terms of Deposit or contact the SDR team with any questions.</p>
+<p>We ask that all SDR depositors review the Terms of Deposit on an annual basis. <a href="https://sdr.library.stanford.edu/documentation/understanding-sdr-terms-deposit">Learn more about the Terms of Deposit</a> or <a href="mailto:sdr-support@lists.stanford.edu">contact the SDR team</a> with any questions.</p>
```

Comment:
This email address is available from `Settings.support_email`.

Reply:
fixed